mlrun/utils/notifications/notification/ipython.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import typing
+
+import mlrun.lists
+import mlrun.utils.helpers
+
+from .base import NotificationBase, NotificationSeverity
+
+
+class IPythonNotification(NotificationBase):
+    def __init__(
+        self,
+        header: str,
+        severity: typing.Union[str, NotificationSeverity] = NotificationSeverity.info,
+        runs: typing.Union[list, mlrun.lists.RunList] = None,
+        params: typing.Dict[str, str] = None,
+        custom_html: str = None,
```

Comment:
lets add description from where each notification class is expected to be called ( e.g from client / server and why )